Unable to install `ember-bootstrap` with Ember 6.4.0 LTS due to peer dependency conflict with `@glimmer/component@2.0.0`

Hi, I'm unable to install `ember-bootstrap` in a brand new Ember 6.4.0 LTS project due to a peer dependency conflict with `@glimmer/component`.

### Steps to Reproduce

```bash
npm install -g ember-cli@6.4.0
ember new frontend --lang en
cd frontend
ember install ember-bootstrap
```

### Error Output

```
Installing packages... This might take a couple of minutes.
Command failed with exit code 1: npm install --save-dev --loglevel error ember-bootstrap
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: frontend@0.0.0
npm ERR! Found: @glimmer/component@2.0.0
npm ERR! node_modules/@glimmer/component
npm ERR!   dev @glimmer/component@"^2.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @glimmer/component@"^1.0.4" from ember-bootstrap@6.5.0
npm ERR! node_modules/ember-bootstrap
npm ERR!   dev ember-bootstrap@"*" from the root project
```

### Environment

* Ember CLI: 6.4.0
* Node: (Your Node version here)
* OS: Windows 

### Notes

It looks like `ember-bootstrap@6.5.0` requires `@glimmer/component@^1.0.4`, which is incompatible with the version included in Ember 6.4.0 (`@glimmer/component@2.0.0`).

Is there a compatible version of `ember-bootstrap` for Ember 6.4, or plans to support `@glimmer/component@2.x`?

Thanks!

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -81,6 +81,8 @@ jobs:
             bootstrap: 5
           - scenario: tracked-toolbox-v1
             bootstrap: 5
+          - scenario: glimmer-component-v1
+            bootstrap: 5
     steps:
       - name: Checkout code
         uses: wyvox/action@v1

--- a/ember-bootstrap/package.json
+++ b/ember-bootstrap/package.json
@@ -98,7 +98,7 @@
     "typescript": "5.9.3"
   },
   "peerDependencies": {
-    "@glimmer/component": "^1.0.4",
+    "@glimmer/component": "^1.0.4 || ^2.0.0",
     "@glimmer/tracking": "^1.0.4",
     "ember-source": ">=4.8.0",
     "ember-modifier": "^3.2.7 || ^4.0.0"

--- a/test-app/config/ember-try.js
+++ b/test-app/config/ember-try.js
@@ -76,6 +76,14 @@ module.exports = async function () {
           },
         },
       },
+      {
+        name: 'glimmer-component-v1',
+        npm: {
+          devDependencies: {
+            '@glimmer/component': '^1.1.2',
+          },
+        },
+      },
       embroiderSafe({
         npm: {
           devDependencies: {

--- a/test-app/tests/integration/components/bs-carousel-test.gts
+++ b/test-app/tests/integration/components/bs-carousel-test.gts
@@ -19,6 +19,7 @@ import { tracked } from '@glimmer/tracking';
 import type { RenderingTestContext } from '@ember/test-helpers';
 import Component from '@glimmer/component';
 import { registerDestructor } from '@ember/destroyable';
+import type Owner from '@ember/owner';
 
 const TRANSITION_DURATION = 50;
 
@@ -567,7 +568,7 @@ module('Integration | Component | bs-carousel', function (hooks) {
     }
 
     class CustomSlide extends Component<CustomSlideSignature> {
-      constructor(owner: unknown, args: CustomSlideSignature['Args']) {
+      constructor(owner: Owner, args: CustomSlideSignature['Args']) {
         super(owner, args);
         this.args.registerChild?.(this);
 

